plugins/node/opentelemetry-instrumentation-restify/package.json

```
@@ -50,8 +50,10 @@
     "@opentelemetry/context-async-hooks": "0.25.0",
     "@opentelemetry/sdk-trace-base": "0.25.0",
     "@opentelemetry/sdk-trace-node": "0.25.0",
+    "@types/bluebird": "^3.5.36",
     "@types/mocha": "7.0.2",
     "@types/node": "14.17.9",
+    "bluebird": "^3.7.2",
```

```suggestion
    "bluebird": "3.7.2",
```
devDependencies are pinned in general